docs/manual/spec.md

```diff
@@ -461,6 +461,13 @@ when name is omitted, the description refers to the main package.
 Package build is divided into multiple separate steps, each executed
 in a separate shell.
 
+Only one of each section can be present in a spec, but all build scriptlets
+except for `%prep` accept options `-a` and `-p`, for append and prepend mode.
+Append and prepend append or prepend lines to the section in the order they
```

Comment:
3 times mentioning "append / prepend" in one place is a bit too much, isn't it? :)

Reply:
Sure. Feel free to suggest a better wording.

Reply:
Hopefully significantly less hideous now.